### DISCUSSION TITLE
Falta de consigna de la descarga de MinGW

### DISCUSSION THREAD

**Discussion by Lautymendez** [INGCOM-UNRN-P1/cursada-2023#3] [Preguntas y respuestas]
Buenas tardes Martin,

Realizando el TP 0 llegue al punto donde hay que descargar el minGW, si bien aparece el link de descarga, no aparece especificado donde se baja el archivo.

Saludos!,

Lautaro Mendez.

**Comment by martinvilu** [comment]
Este es para aquellos que quieran tener todo instalado en su computadora, aquellos que necesiten tener ‘flexibilidad’ y tener las herramientas en un pendrive, tendrán que esperar a que lo suba para su descarga.

Seguí la guía que está en: https://github.com/INGCOM-UNRN-P1/cursada-2023/wiki/Instalaci%C3%B3n-de-las-herramientas

La descarga en sí es la siguiente: [winlibs_mingw](https://github.com/brechtsanders/winlibs_mingw/releases/download/13.2.0-16.0.6-11.0.0-ucrt-r1/winlibs-x86_64-posix-seh-gcc-13.2.0-llvm-16.0.6-mingw-w64ucrt-11.0.0-r1.zip)
